src/libs/collect.c

```diff
@@ -605,6 +605,7 @@ static gboolean view_onMouseScroll(GtkWidget *treeview, GdkEventScroll *event, d
     gtk_widget_get_size_request(GTK_WIDGET(d->scrolledwindow), &width, &height);
     height = height + increment*event->delta_y;
     height = (height < min_height) ? min_height : (height > max_height) ? max_height : height;
+    printf("view_onMouseScroll height %d\n", height);
```

Comment:
you probably want to remove this debug code :)

Reply:
for sure !